apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx

```diff
@@ -36,25 +46,56 @@ export const TicketChange: React.FC<ITicketChangeProps> = ({ ticketChange }) =>
 
 const useChangedFieldMessagesOf = (ticketChange) => {
     const intl = useIntl()
-    const ClientPhoneMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.clientPhone' })
-    const DetailsMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.details' })
-    const ClientNameMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.clientName' })
-    const IsPaidMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.isPaid' })
-    const IsEmergencyMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.isEmergency' })
-    const StatusDisplayNameMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.statusDisplayName' })
-    const UnitNameMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.unitName' })
-    const AssigneeMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.assignee' })
-    const ClassifierMessage = intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.classifier' })
-    const fields = [
-        ['clientPhone', ClientPhoneMessage],
-        ['details', DetailsMessage],
-        ['clientName', ClientNameMessage],
-        ['isPaid', IsPaidMessage],
-        ['isEmergency', IsEmergencyMessage],
-        ['statusDisplayName', StatusDisplayNameMessage],
-        ['unitName', UnitNameMessage],
-        ['assigneeDisplayName', AssigneeMessage],
-        ['classifierDisplayName', ClassifierMessage],
+
+    const getChangeMessages = (baseId: string) => {
+        return {
+            add: intl.formatMessage({ id: `${baseId}.add` }),
+            change: intl.formatMessage({ id: `${baseId}.change` }),
+            remove: intl.formatMessage({ id: `${baseId}.remove` }),
+        }
+    }
+
+    const fields: ITicketChangeField[] = [
```

Comment:
we have a style convention about `intl` it should be at the top of the component.